backend/core/src/units-summary/entities/units-summary.entity.ts

```diff
@@ -53,13 +53,13 @@ class UnitsSummary {
   @Expose()
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
   @IsNumberString({}, { groups: [ValidationsGroupsEnum.default] })
-  minimumIncomeMin?: string | null
+  monthlyIncomeMin?: string | null
 
   @Column({ nullable: true, type: "text" })
   @Expose()
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
   @IsNumberString({}, { groups: [ValidationsGroupsEnum.default] })
-  minimumIncomeMax?: string | null
+  monthlyIncomeMax?: string | null
```

Comment:
These renames are to put it in line with the information existing in the Units table?

Reply:
That's correct, it's much easier to deal with generics when the fields have the same name (otherwise I have to add both names and handle each case).

Reply:
My understanding was that renaming these fields was mostly about making it easier to transform `UnitsSummary` into an instance of `UnitsSummarized` - if that's right, we could revert these changes if we want? (I don't have strong feelings, though)

Reply:
I thought it may be useful to rename in case we use generics in some of the ui components, so we can support both data structures, but I think you're right. It's probably best to make those changes when necessary, in a different PR.